lexer.py

```
@@ -0,0 +1,100 @@
+        self.state: Callable = None
+        self.position: int = -1
+        self.start: int = 0
+        self.paren_depth: int = 0
+        self.brace_depth: int = 0
+        self.items: list = []
+
+    def get( self ) -> str:
+        if int( self.position ) >= len( self.input ) - 1:
+            self.position += 1
+            return end
+        else:
```

Ruff (0.11.9)

Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)